week_1/String/group_anagrams/group_anagrams.test.js

```diff
@@ -0,0 +1,23 @@
+const groupAnagrams = require("./group_anagrams");
+
+describe("group_anagrams", () => {
+	test("works with one string", () => {
+		expect(groupAnagrams(["hey"])).toEqual([["hey"]]);
```

Comment:
This doesn't feel right. "hey" isn't an anagram of "hey", is it? Feels like if the array has a length of 1, it should be filtered out.

Reply:
I updated the logic and tests so that a word can't be an anagram of itself.